### PR TITLE
Merge paradox/scala/agents.md and java/agents.md

### PR DESCRIPTION
**Pull request by richardimaoka**
Refs #23052

## Rendered HTML:

- https://richard-akka-doc.firebaseapp.com/scala/agents.html
- https://richard-akka-doc.firebaseapp.com/java/agents.html

## I split this in two commits:

- **1st commit:** Identical contents in scala/agents.md and java/agents.md -> **this is to take the diff both in scala/actors.md and java/agents.md**

  - [scala/agents.md  (prev) vs. scala/agents.md (after)](https://github.com/richard-imaoka/akka/commit/47ccd42b51c3d28b21d2c3ed778d7476056e5398#diff-34b7059981f1dbfe1d0a29840efa6a51)
  - [java/agents.md (prev) vs. scala/agents.md (after)](https://github.com/richard-imaoka/akka/commit/47ccd42b51c3d28b21d2c3ed778d7476056e5398#diff-4bf0ef2fcfac6ef5dfd8df7eca25c373)

- **2nd commit:** Replace java/agents.md with a symlink to scala/agents.md

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/agents.md [hunk 1]
```diff
@@ -38,53 +38,80 @@ read or update the Agent.
...
 state of an Agent is synchronous.
 
+@@@ div { .group-java }
+
+You can also get a `Future` to the Agents value, that will be completed after the
+currently queued updates have completed:
+
+@@snip [AgentDocTest.java]($code$/java/jdocs/agent/AgentDocTest.java) { #import-future #read-future type=java }
+
+See @ref:[Futures](futures.md) for more information on `Futures`.
```
Comment (raboof) [inline akka-docs/src/main/paradox/scala/agents.md]: Perhaps spell this "`Future`s"?

**Review by raboof — APPROVED** [review]
LGTM,  thanks!

File: akka-docs/src/main/paradox/scala/agents.md (hunk 1 above)
Comment (richard-imaoka) [inline akka-docs/src/main/paradox/scala/agents.md]: Sure, updated now.

**Comment by raboof** [comment]
Lovely, thanks!

**Pull request merged**